src/uncrustify.cpp

```diff
@@ -1507,8 +1509,11 @@ static bool bout_content_matches(const file_mem &fm, bool report_status)
    if (  is_same
       && report_status)
    {
-      fprintf(stdout, "PASS: %s (%u bytes)\n",
-              cpd.filename.c_str(), static_cast<int>(fm.raw.size()));
+      if (is_quiet)
```

Comment:
Shouldn't this be `!is_quiet`? when it is quiet, we should not print the message.

Reply:
Hmmm, you're right. Unsure why, but the above gave the right behavior for some reason. I'll investigate, I have a suspicion what the problem is.

Reply:
OK, I think I've fixed it now.